EDEngineer/Resources/Data/localization.json

```diff
@@ -1718,12 +1718,6 @@
       "de": null,
       "fr": null,
       "ru": null
-    },

```

Comment:
good job ! just please do not remove that last block, I added it a short while ago (it was the last part of the application that it wasn't yet possible to translate)